### PR TITLE
minor tweak to ignore trailing numbers

### PR DESCRIPTION
**Pull request by 927589452**
fixes usage of foldlevel for #1438

### CODE REVIEW

File: autoload/vimtex/fold/markers.vim [hunk 1]
```diff
@@ -17,7 +17,7 @@ let s:folder = {
       \   'start' : '\v\%.*\{\{\{',
       \   'end' : '\v\%\s*\}\}\}',
       \   'text' : [
-      \     ['\v\%\s*\{\{\{\s*\zs.*', '% {{{ '],
+      \     ['\v\%\s*\{\{\{[0-9]?\s*\zs.*', '% {{{ '],
```
Comment (lervag) [inline autoload/vimtex/fold/markers.vim]: You can use `\d` instead of `[0-9]`.

**Comment by lervag** [comment]
Thanks for noticing and suggesting a fix! I had a minor comment, will merge when it is updated.

File: autoload/vimtex/fold/markers.vim (hunk 1 above)
Comment (927589452) [reply autoload/vimtex/fold/markers.vim]: thanks, didn't have this in mind
should I rewrite it?

Comment (lervag) [reply autoload/vimtex/fold/markers.vim]: Yes, please.

**Comment by 927589452** [comment]
sorry for taking so long; thought i already did it

**Comment by lervag** [comment]
Thanks!

**Pull request merged**